tests/filecheck/dialects/riscv/riscv_ops_custom_syntax.mlir

```diff
@@ -0,0 +1,270 @@
+  %slti = "riscv.slti"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %sltiu = "riscv.sltiu"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %andi = "riscv.andi"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %ori = "riscv.ori"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %xori = "riscv.xori"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %slli = "riscv.slli"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %srli = "riscv.srli"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %srai = "riscv.srai"(%0) {"immediate" = 1 : i32}: (!riscv.reg<>) -> !riscv.reg<>
+  %lui = "riscv.lui"() {"immediate" = 1 : i32}: () -> !riscv.reg<>
+  %auipc = "riscv.auipc"() {"immediate" = 1 : i32}: () -> !riscv.reg<>
+  %mv = "riscv.mv"(%0) : (!riscv.reg<>) -> !riscv.reg<>
+  // CHECK-NEXT: %addi = riscv.addi %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %slti = riscv.slti %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %sltiu = riscv.sltiu %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %andi = riscv.andi %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %ori = riscv.ori %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %xori = riscv.xori %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %slli = riscv.slli %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %srli = riscv.srli %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %srai = riscv.srai %0 1 : !riscv.reg<>
+  // CHECK-NEXT: %lui = riscv.lui 1 : !riscv.reg<>
+  // CHECK-NEXT: %auipc = riscv.auipc 1 : !riscv.reg<>
+  // CHECK-NEXT: %mv = riscv.mv %0 : !riscv.reg<>
+
+  // Integer Register-Register Operations
+  %add = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %slt = "riscv.slt"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sltu = "riscv.sltu"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %and = "riscv.and"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %or = "riscv.or"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %xor = "riscv.xor"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sll = "riscv.sll"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %srl = "riscv.srl"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sub = "riscv.sub"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %sra = "riscv.sra"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  "riscv.nop"() : () -> ()
+  // CHECK-NEXT: %add = riscv.add %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %slt = riscv.slt %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sltu = riscv.sltu %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %and = riscv.and %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %or = riscv.or %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %xor = riscv.xor %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sll = riscv.sll %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %srl = riscv.srl %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sub = riscv.sub %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: %sra = riscv.sra %0, %1 : !riscv.reg<>
+  // CHECK-NEXT: riscv.nop :
```

Comment:
We should try to drop the `:` here, if there are no types behind...

Reply:
```:``` signals the end of attribute, not the start of types.